lua/kubectl/views/jobs/init.lua

```diff
@@ -0,0 +1,80 @@
+local ResourceBuilder = require("kubectl.resourcebuilder")
+local buffers = require("kubectl.actions.buffers")
+local commands = require("kubectl.actions.commands")
+local definition = require("kubectl.views.jobs.definition")
+local informer = require("kubectl.actions.informer")
+local tables = require("kubectl.utils.tables")
+
+local M = {
+  builder = nil,
+}
+
+function M.View(cancellationToken, from_cronjob)
+  if M.builder then
+    M.builder = M.builder
+        :display(definition.ft, definition.display_name, cancellationToken)
+        :setCmd(definition.url, "curl")
+        :fetchAsync(function(builder)
+          builder:decodeJson()
+          informer.start(builder)
+          vim.schedule(function()
+            M.Draw(cancellationToken, from_cronjob)
+          end)
+        end)
+  else
+    M.builder = ResourceBuilder:new(definition.resource)
+        :display(definition.ft, definition.display_name, cancellationToken)
+        :setCmd(definition.url, "curl")
+        :fetchAsync(function(builder)
+          builder:decodeJson()
+          informer.start(builder)
+          vim.schedule(function()
+            M.Draw(cancellationToken, from_cronjob)
+          end)
+        end)
+  end
+end
+
+function M.Draw(cancellationToken, from_cronjob)
+  M.builder = M.builder
+  local data = M.builder.data
+  if from_cronjob then
+    local filtered_data = {}
+    for _, item in ipairs(data.items) do
+      if item.metadata.namespace == from_cronjob.ns and item.metadata.ownerReferences and item.metadata.ownerReferences[1].name == from_cronjob.name then
+        table.insert(filtered_data, item)
+      end
+    end
+    M.builder.data = { items = filtered_data }
+  end
+
+  M.builder = M.builder
+      :process(definition.processRow):sort():prettyPrint(definition.getHeaders)
+
+  M.builder = M.builder:addHints(definition.hints, true, true, true):setContent(cancellationToken)
+end
+
+function M.Desc(name, ns)
+  ResourceBuilder:new("desc")
+      :displayFloat("k8s_job_desc", name, "yaml")
```

Comment:
we have the Resourcebuilder.view_float helper if we aren't doing anything special here

Reply:
I just copied it from every other view:
https://github.com/mosheavni/kubectl.nvim/blob/25f7daafe10497d756fa12d9f11c6a27b8e6be3e/lua/kubectl/views/crds/init.lua#L30-L37
https://github.com/mosheavni/kubectl.nvim/blob/25f7daafe10497d756fa12d9f11c6a27b8e6be3e/lua/kubectl/views/configmaps/init.lua#L32-L39

and many more

i think, if you want to change it, we can create an issue for changing it to EVERYTHING, not just this one.

Reply:
I think both approaches are valid,  i.e. all new code uses the new way  vs doing a big refactor at once but no let's merge this sucker!!